R/pars.R

```diff
@@ -188,6 +188,8 @@ spim_pars_beta <- function(date, last_beta_days_ago = 21) {
   ##                  (help the model stabilise a long period of time)
   ## 28. 2021-10-01 - Point before recent increase in cases - hospitalisations
   ## 29. 2021-10-22 - School half-term - Point before recent plateau in cases
+  ## 30. 2021-11-01 - Schools return
+  ## 31. 2021-12-06 - Pragmatic date - Omicron transmission increases
```

Comment:
What's the logic of this date?

Reply:
Would 2021-12-08 be better (announcement of move to Plan B)? https://www.gov.uk/government/news/prime-minister-confirms-move-to-plan-b-in-england

Reply:
Good shout!